packages/deployer/src/deployUtils.ts

```diff
@@ -151,3 +160,99 @@ export async function printRelayInfo (hre: HardhatRuntimeEnvironment): Promise<v
   console.log(chalk.white('Relayer register:'))
   console.log(chalk.grey(`gsn relayer-register -m $SECRET --network ${network.url} --relayUrl https://${hre.hardhatArguments.network}.v3.opengsn.org/v3 --token ${stakingTokenAddress} --stake ${stakingTokenValue} --wrap`))
 }
+
+export async function getEnv (hre: HardhatRuntimeEnvironment): Promise<{ deployer: string, deployments: DeploymentsExtension, env: Environment }> {
```

Comment:
We have `getEnvironment` global function. Please rename to `getDeploymentEnvironment`.